### PR TITLE
OSDOCS-7075-nutanix: Added user-managed LB to Nutanix docs

### PR DESCRIPTION
**Pull request by dfitzmau**
Version(s):
4.16

Issue:
* [OSDOCS-7075](https://issues.redhat.com/browse/OSDOCS-7075)

Link to docs preview:
* [BM DAY 0](https://76524--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_bare_metal_ipi/ipi-install-installation-workflow.html#nw-osp-services-external-load-balancer_ipi-install-installation-workflow)
* [BM DAY 2](https://76524--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_bare_metal_ipi/ipi-install-post-installation-configuration.html#nw-osp-services-external-load-balancer_ipi-install-post-installation-configuration)
* [Nutanix](https://76524--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_nutanix/installing-nutanix-installer-provisioned.html#nw-osp-services-external-load-balancer_installing-nutanix-installer-provisioned)
* [vSphere restricted](https://76524--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/ipi/installing-restricted-networks-installer-provisioned-vsphere.html#nw-osp-services-external-load-balancer_installing-restricted-networks-installer-provisioned-vsphere)
* [vSphere customizations](https://76524--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/ipi/installing-vsphere-installer-provisioned-customizations.html#nw-osp-services-external-load-balancer_installing-vsphere-installer-provisioned-customizations)
* [vSphere network customizations](https://76524--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/ipi/installing-vsphere-installer-provisioned-network-customizations.html#nw-osp-services-external-load-balancer_installing-vsphere-installer-provisioned-network-customizations)
* [OpenStack](https://76524--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/load-balancing-openstack.html#nw-osp-services-external-load-balancer_load-balancing-openstack)

- [x] SME has approved this change.
- [x] QE has approved this change.

Additional information:

* I removed the user-managed LB docs for the bare-metal day 2 docs because of support issues. Although it might work with no issues, the use-case as a day 2 task has not been fully tested.

### CODE REVIEW

File: modules/nw-osp-configuring-external-load-balancer.adoc [hunk 1]
```diff
@@ -1,10 +1,12 @@
 // Module included in the following assemblies:
 
-// OpenStack
-// * networking/load-balancing-openstack.adoc
 // Bare metal
 // * installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc
 // * installing/installing_bare_metal_ipi/ipi-install-post-installation-configuration.adoc
+// OpenStack
+// * networking/load-balancing-openstack.adoc
+// Nutanix
```
Comment (dfitzmau) [inline modules/nw-osp-configuring-external-load-balancer.adoc:8]: Do I need to specify that a user-managed LB is possible as a Day-2 task for Nutanix and OpenStack?

Comment (zhaozhanqi) [reply modules/nw-osp-configuring-external-load-balancer.adoc:8]: hmm, I don't think we need to add the day2 task if no before.

File: modules/nw-osp-services-external-load-balancer.adoc [hunk 2]
```diff
@@ -33,6 +39,14 @@ Red Hat supports the following services for a user-managed load balancer:
 * OpenShift API
 * OpenShift MachineConfig API
 
+
+ifdef::post-bare-metal[]
```
Comment (cybertron) [inline modules/nw-osp-services-external-load-balancer.adoc:43]: I would still prefer to just remove this from the post-install docs. We expect the new user-managed loadbalancer mechanism to replace the old post-install one. If it turns out a lot of customers still want to use the old way for some reason we can always put it back.

Comment (dfitzmau) [reply modules/nw-osp-services-external-load-balancer.adoc]: Thanks, removed.

**Review by skopacz1 — COMMENTED** [review]
Left just a few comments, otherwise LGTM!

**Review by cybertron — COMMENTED** [review]
/lgtm

**Review by skopacz1 — APPROVED** [review]
Merge review LGTM

**Pull request merged**